content/docs/reference/spec/migration/buildpack-api-0.4-0.5.md

```diff
@@ -0,0 +1,39 @@
++++
+title="Buildpack API 0.4 -> 0.5"
++++
+
+<!--more-->
+
+This guide is most relevant to stack and buildpack authors. 
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.5) for buildpack API 0.5 for the full list of changes and further details.
+
+## Stack author
+
+### Stack ID
+Stack ID requirements have been removed from the buildpack specification.
+Each stack must indicate either a compatible stack and provide a valid stack ID, or compatibility with any stack by specifying the special value `*`.
+ 
+## Buildpack author
+
+### Character restrictions for process types
+`type` has been restricted to only contain numbers, letters, and the characters ., _, and -, and now symlinks on both linux and windows filesystems can be created.
```

Comment:
It would be nice to specify the file in which this key exists.

Reply:
Please see my changes in [this commit](https://github.com/buildpacks/docs/pull/279/commits/85fb1f1db86cb9637d7e6569af61afb42c91f00d).
I'll resolve this conversation. Please feel free to reopen it if you think that something is still missing.